tests/cypress/common/views.js

```diff
@@ -1169,3 +1169,19 @@ export const verifyPolicyTemplateViolationDetailsForCluster = (policyName, polic
     }
   })
 }
+  export const checkPolicyNoResourcesExist = (valid_policy=true, message='Resource not found') => {
```

Comment:
Please make this a cypress command. I guess I could use it also here to verify there are no policies present on the cluster, what do you think?
https://github.com/open-cluster-management/grc-ui/blob/main/tests/cypress/tests/all_policies_summary_table.spec.js#L16

Also, if this is really checking the presence/absence of the respective graphics on the page. I would propose renaming the command to something like `checkNoResourceIcon(present=true|false, message='Resource not found')`
And finally, please add at least single line comment describing the purpose of the command

Reply:
@kkaarreell This function is not only checking icon but also the message. I'll make requested changes